### ISSUE TITLE
Test coverage for unrecognized methods (both service-side and invocation-side)

### ISSUE THREAD

**Issue by endobson** [grpc/grpc#2002]
I got the following from the python server.

```
ERROR:root:Unexpected exception from task run in logging pool!
Traceback (most recent call last):
  File "/Users/endobson/proj/grpc/python2.7_virtual_environment/lib/python2.7/site-packages/grpc/framework/foundation/logging_pool.py", line 43, in _wrapping
    return behavior(*args, **kwargs)
  File "/Users/endobson/proj/grpc/python2.7_virtual_environment/lib/python2.7/site-packages/grpc/_adapter/fore.py", line 231, in _spin
    self._on_service_acceptance_event(event, server)
  File "/Users/endobson/proj/grpc/python2.7_virtual_environment/lib/python2.7/site-packages/grpc/_adapter/fore.py", line 125, in _on_service_acceptance_event
    self._request_deserializers[method],
KeyError: 'Method'
```

My client is currently just sending an empty initial metadata and asking for a method "Method". I only get this once so I assume that means that the processing thread died which seems bad.

**Comment by nathanielmanistaatgoogle** [comment]
Thanks for the report; I agree that this looks like a problem.

In what language is the client with which you're making these requests?

**Comment by endobson** [comment]
Racket. I'm writing my own client on top of the C bindings.

**Comment by nathanielmanistaatgoogle** [comment]
Cool!

Were you using the Python client you would have seen this problem client-side.

**Comment by nathanielmanistaatgoogle** [comment]
(Changed the bug title to capture this defect report's exit condition.)

**Comment by nathanielmanistaatgoogle** [comment]
This defect is specific to the alpha code and will be resolved this week as the beta code goes live.

**Comment by jayantkolhe** [comment]
Marking fixed

**Comment by nathanielmanistaatgoogle** [comment]
I wouldn't call this fixed today but I will call it subsumed by #2634, so consider it "resolved as duplicate". :-)
